### PR TITLE
Updated doctrine/coding-standard to 6.0

### PR DESCRIPTION
**Pull request by patrickjahns**
Changed `doctrine/coding-standard` in `composer.json` to `6.0`  and fixed found issues via `phpcbf`

### CODE REVIEW

File: Twig/DoctrineExtension.php [hunk 1]
```diff
@@ -290,6 +290,7 @@ public function replaceQueryParameters($query, $parameters)
             '/\?|((?<!:):[a-z0-9_]+)/i',
             static function ($matches) use ($parameters, &$i) {
                 $key = substr($matches[0], 1);
+
```
Comment (patrickjahns) [inline Twig/DoctrineExtension.php]: This seems to be related to https://github.com/slevomat/coding-standard/issues/635

Will confirm and see if we can fix the upstream issue

**Comment by patrickjahns** [comment]
Travis failures are related to deprecation notices from TWIG - not sure how to progress here

**Comment by SenseException** [comment]
Deprecations because of new Twig namespaces are handled in #929.

**Comment by alcaeus** [comment]
Thanks @patrickjahns!

**Pull request merged**